cmd/akashd/init.go

```diff
@@ -5,10 +5,11 @@ import (
 	"github.com/ovrclk/akash/types/base"
 	"github.com/ovrclk/akash/util/initgen"
 	"github.com/spf13/cobra"
+	"math"
 )
 
 const (
-	maxTokens uint64 = 1000000000
+	maxTokens uint64 = math.MaxUint64
```

Comment:
I think this will screw up the integration tests [here](https://github.com/ovrclk/akash/blob/master/_integration/cmp/account.go#L28).

* Simple matter of changing the value in the test or ideally just referencing a constant.
* The json lib we're using there maxes out at `int64` which won't be able to hold `math.MaxUint64`.
* Also, I think we're supposed to have a billion tokens total.

Reply:
Yeah it read it as `-1`:

```
Evaluation of main failed:
/main/account-send/account-balance/akash: ../akash -d ./data/client query account 22CDFF3E908986DE2F4B1D279667BA02B84CD7D6: balance: received -1 expected 1000000000
-===[BEGIN STDOUT]===-
{
  "address": "22CDFF3E908986DE2F4B1D279667BA02B84CD7D6",
  "balance": 18446744073709551615
}
```